Makefile

```diff
@@ -25,6 +25,7 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
+TEST_TAGS ?= basic
```

Comment:
@hferentschik so looks like if user doesn't define a tag during run of integration then it only consider to run basic tests but I do anticipate it should be running all the test instead of basic when no tags are defined.